pageserver/src/storage_sync2.rs

```diff
@@ -540,6 +560,10 @@ impl RemoteTimelineClient {
             let upload_queue = guard.initialized_mut()?;
             if let Some(upgraded) = upload_queue.latest_files.get_mut(layer_file_name) {
                 upgraded.merge(&new_metadata);
+                // We don't know when the next index part upload will happen.
+                // So, account for this eagerly.
+                // XXX for predictability of the metric, would it be better to omit this?
```

Comment:
Opinions on this?

Reply:
Presumably, ok, since we have to download those layers anyway and on restart this value will be repopulated.

Reply:
>  and on restart this value will be repopulated.

If we don't do an index file upload inbetween here and restart, the value will go back down after restart.

Reply:
works for me